code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -133,6 +133,18 @@
 	required_reagents = list("slime_toxin" = 1, "mutagen" = 1)
 
 
+/datum/chemical_reaction/fermis_plush
+	name = "Fermis plush"
+    id = "fermis_plush"
+    required_reagents = list("sugar" = 10, "mutagen" = 10)
```

Comment:
Recipe will cause issues with smoke grenades that have chem recipes relying on mutagen.

Reply:
oops